acceptance-test/src/acceptance-test/java/org/zalando/nakadi/webservice/BinaryEndToEndAT.java

```diff
@@ -0,0 +1,42 @@
+package org.zalando.nakadi.webservice;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.zalando.nakadi.domain.EnrichmentStrategyDescriptor;
+import org.zalando.nakadi.domain.EventCategory;
+import org.zalando.nakadi.utils.EventTypeTestBuilder;
+import org.zalando.nakadi.webservice.utils.NakadiTestUtils;
+
+import java.util.Base64;
+import java.util.List;
+
+import static com.jayway.restassured.RestAssured.given;
+
+public class BinaryEndToEndAT extends BaseAT {
+    private static final String TEST_ET_NAME = "nakadi.test-2022-05-06.et";
+    private static final String TEST_DATA = "BAAAAGO0v/qJk2BIZjU5ZmVlNTQtMmNhYy00MTAzLWI4NTItOGMwOGRiZjhlNjEyAhJ0ZX" +
+            "N0LWZsb3cAAjECEnRlc3QtdXNlcjJuYWthZGkudGVzdC0yMDIyLTA1LTA2LmV0AAAAAAAAAABBCFBPU1QYL2V2ZW50LXR5cGVzAB50ZX" +
+            "N0LXVzZXItYWdlbnQMbmFrYWRpFGhhc2hlZC1hcHCSAxQCLQQtLfYBggU=";
+
+    @Test
+    @Ignore
+    public void testAvroPublishing() throws JsonProcessingException {
+        final var et = EventTypeTestBuilder.builder()
+                .name(TEST_ET_NAME)
+                .category(EventCategory.BUSINESS)
+                .enrichmentStrategies(List.of(EnrichmentStrategyDescriptor.METADATA_ENRICHMENT))
+                .build();
+        NakadiTestUtils.createEventTypeInNakadi(et);
+
+        final byte[] body = Base64.getDecoder().decode(TEST_DATA);
```

Comment:
the endpoint is not suppose to accept base64, only binary payload

Reply:
Yes, only binary data is send to the endpoint. 
Instead of creating the binary data in the test (using schema) for every test execution, I created it once locally. The binary data that I created locally is encoded in base64 so that it can be included as a string in the test. And this is decoded in the above line so that we send binary data to the endpoint.